src/context/TaskContextProvider.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react'
+import TaskContext from './TaskContext'
+import axios from 'axios';
+
+const TaskContextProvider = ({ children }) => {
+
+    const [tasks, setTasks] = useState([]);
+    const [taskName, setTaskName] = useState('');
```

Comment:
Why are we making separate states for theses fields? When we can make an object consisting of taskName, description etc.